src/types.ts

```diff
@@ -10,3 +10,40 @@ export interface InitializerConfig {
   alias: string
   birthdayHeight: number
 }
+
+export interface SpendInfo {
+  zatoshi: string
+  toAddress: string
+  memo: string
+  fromAccountIndex: number
+  spendingKey?: string
+}
+
+export interface TransactionQuery {
+  offset?: number
+  limit?: number
+  startDate?: number
+  endDate?: number
+}
+
+export interface ZcashTransaction {
+  txId: string
+  accountIndex: number
+  fee: string
+  value: string
+  direction: 'inbound' | 'outbound'
+  toAddress: string
+  memo?: string
+  minedHeight: number // 0 for unconfirmed
+  blockTime: number // UNIX timestamp
+}
+
+export type PendingTransaction = ZcashTransaction & {
+  accountIndex: number
```

Comment:
accountIndex is already in ZcashTransaction

Reply:
Good catch, I think this was a result of [uncommenting code](https://github.com/EdgeApp/react-native-zcash/commit/baa52dd2dcfd74038a4913bda4cc97685e52370c#diff-2eb03c79436b0e9801fde914633f0e5259d161ac3436a5b6fd50c5691dd5c4ccR52-R70) after applying [changes from the gist](https://gist.github.com/paullinator/c37efc137f50fd7e11e6e4ffca7b0a94#gistcomment-3451752).

Addressed in commit d1ff79a